src/state/users.ts

```diff
@@ -6,22 +6,14 @@ type UsersDB = { [userId: number]: string };
 export type UserAction = string;
 type UserActionError = string;
 
-// interface SetUserName {
-//   type: "SetUserName";
-//   username: string;
-// }
-
-const usersName = `${appName}/user-settings`;
+const usersName = `${appName}/user-settings-1`;
```

Comment:
I had to change the name so that I could edit the reducer. It's a super weird (bad) convention, but it kinda works for now...